Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -163,13 +163,17 @@ extension MongoCollection {
      *   - `ServerError.writeError` if an error occurs while performing the write.
      *   - `ServerError.commandError` if an error occurs that prevents the command from executing.
      *   - `UserError.invalidArgumentError` if the options passed in form an invalid combination.
+     *   - `UserError.logicError` if the provided session is inactive.
      *   - `EncodingError` if an error occurs while encoding the index specification or options.
      */
     @discardableResult
     public func createIndex(_ keys: Document,
                             options: IndexOptions? = nil,
-                            commandOptions: CreateIndexOptions? = nil) throws -> String {
-        return try createIndexes([IndexModel(keys: keys, options: options)], options: commandOptions)[0]
+                            commandOptions: CreateIndexOptions? = nil,
```

Comment:
I noticed this was called `commandOptions` here, but just `options` in the other index methods. Was that intentional?

Reply:
Oh whoops, I didn't see the options parameter in there as well. Makes sense.